app/views/carts/index.html.erb

```diff
@@ -21,11 +21,11 @@
   <% end %>
   <p><b>Total: $<%= sum %></b></p>
 </div>
-<div>
-  <% if current_user %>
-  <%= link_to "Checkout", dashboard_index_path %>
+<div class="checkout">
+    <% if current_user %>
```

Comment:
I like this name @mimilettd. Keeping our classes properly named goes a long way toward keeping things adaptive and legible.